app-config.yaml

```diff
@@ -97,3 +97,19 @@ auth:
       development:
         clientId: ${AUTH_GITHUB_CLIENT_ID}
         clientSecret: ${AUTH_GITHUB_CLIENT_SECRET}
+
+
+kubernetes:
```

Comment:
Please move this to `app-config.prod.yaml` and create similar in `app-config.local.yaml` for locally connected cluster (via `localKubectlProxy`) and `app-config.dev.yaml` using local cluster api url (`openshift.default.svc.cluster.local`).

Reply:
Didn't know about the local cluster api url, thanks for letting me know! Changed in the latest commit.